iptables/parameters/protocol.go

```diff
@@ -75,10 +77,22 @@ func destinationPort(port uint16, negative bool) *TcpUdpParameter {
 	}
 }
 
+func destinationPortRangeOrValue(uIDsToPorts config.UIDsToPorts) *TcpUdpParameter {
+	return &TcpUdpParameter{
+		long:  "--destination-port",
+		short: "--dport",
+		value: string(uIDsToPorts.Ports),
+	}
+}
+
 func DestinationPort(port uint16) *TcpUdpParameter {
 	return destinationPort(port, false)
 }
 
+func DestinationPortRangeOrValue(uIDsToPorts config.UIDsToPorts) *TcpUdpParameter {
+	return destinationPortRangeOrValue(uIDsToPorts)
+}
+
```

Comment:
The same suggestion here

```suggestion
func DestinationPort(port uint16) *TcpUdpParameter {
	return destinationPort(port, false)
}

func DestinationPortRangeOrValue(uIDsToPorts config.UIDsToPorts) *TcpUdpParameter {
	return &TcpUdpParameter{
		long:  "--destination-port",
		short: "--dport",
		value: string(uIDsToPorts.Ports),
	}
}

```